core/equations.gms

```diff
@@ -888,17 +888,18 @@ q_changeProdStartyearCost(t,regi,te)$( (t.val gt 2005) AND (t.val eq cm_startyea
 ***---------------------------------------------------------------------------
 *' The use of early retirement is restricted by the following equations:
 ***---------------------------------------------------------------------------
-q_limitCapEarlyReti(ttot,regi,te)$(ttot.val lt 2109 AND pm_ttot_val(ttot+1) ge max(2010, cm_startyear))..
-        vm_capEarlyReti(ttot+1,regi,te)
+q_limitCapEarlyReti(ttot,regi,te)$(ttot.val lt 2120 AND pm_ttot_val(ttot) ge max(2010, cm_startyear)).. !! 2000 doesn't have capacity, so for cm_startyear = 2005 the equation should not be applied
```

Comment:
Do you want to have early retirement also in 2110? Or `ttot` in this equation should be applied only for 2010 to 2100?
If it is the first, I would change the `ttot.val lt 2120` to `ttot.val le 2100`.

PS: Besides removing the 2110 time step, I would use also equality conditionals, as at least in my opinion they are more straight forward to see the affected years in the equation.

Reply:
Thanks for checking this. 

I agree that this could be shifted to "le 2100".

I don't have an opinion about this, whether it is better to run until 2100 or 2110 - I don't think it matters, neither for the results nor the runtime (which was the reason why it was limited at all). I know it shouldn't run longer than 2110 as then it would collide with this line in core/bounds.gms:
 `vm_capEarlyReti.up(ttot,regi,te)$(ttot.val lt 2009 or ttot.val gt 2111) = 0;`
 
 If I hadn't merged already I would change as you propose, but now I guess I wouldn't open a new PR. Or do you feel strongly about it?

Reply:
I don't have strong feelings about it. But maybe it would be nice to have the change added just to keep consistence. 
Maybe you could create an issue pointing to this, so the next person that messes up with the early retirement code could take that into account for any future changes.

Reply:
> I don't have strong feelings about it. But maybe it would be nice to have the change added just to keep consistence. Maybe you could create an issue pointing to this, so the next person that messes up with the early retirement code could take that into account for any future changes.

I put it in my next PR: https://github.com/remindmodel/remind/pull/1295